<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leveledit</class>
 <widget class="QWidget" name="leveledit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>602</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>602</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Level edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../_resources/editor.qrc">
    <normaloff>:/lvl16.png</normaloff>
    <normalon>:/lvl16.png</normalon>
    <activeoff>:/lvl16.png</activeoff>
    <activeon>:/lvl16.png</activeon>:/lvl16.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="interactive">
      <bool>true</bool>
     </property>
     <property name="dragMode">
      <enum>QGraphicsView::RubberBandDrag</enum>
     </property>
     <property name="cacheMode">
      <set>QGraphicsView::CacheBackground</set>
     </property>
     <property name="viewportUpdateMode">
      <enum>QGraphicsView::MinimalViewportUpdate</enum>
     </property>
     <property name="rubberBandSelectionMode">
      <enum>Qt::IntersectsItemShape</enum>
     </property>
     <property name="optimizationFlags">
      <set>QGraphicsView::DontAdjustForAntialiasing|QGraphicsView::DontSavePainterState</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../_resources/editor.qrc"/>
 </resources>
 <connections/>
</ui>
